ZukoAzula2016/src/org/team1540/zukoazula/AutonomousModeLowBar.java

```diff
@@ -5,7 +5,7 @@
 
 public class AutonomousModeLowBar extends AutonomousBase {
 
-    @Tunable(5.0f)
+    @Tunable(18.0f)
```

Comment:
Maybe. Or maybe just halved? Or maybe the new tick count of 100 just needs to be doubled.